README.adoc

```diff
@@ -137,18 +137,19 @@ dynamic configuration with MicroProfile Configuration is used so that you can ea
 or the other down for maintenance. If you want to learn more about setting up dynamic configuration,
 see https://openliberty.io/guides/microprofile-config.html[Configuring microservices].
 
-To enable fault tolerance in the application, create the `SystemResource` class in the
+First, create the `SystemResource` class in the
 `src/main/java/io/openliberty/guides/system/SystemResource.java` file:
 
 [source, java]
 ----
 include::finish/src/main/java/io/openliberty/guides/system/SystemResource.java[tags=503_response]
 ----
 
-The service invokes the `getProperties()` method to display the system properties for a specific
-host. The host displays only if you set the `io_openliberty_guides_system_inMaintenance` configuration
-to `false` in the `finish/resource/CustomConfigSource.json` file. Otherwise, the service throws an
-exception, which makes it unavailable.
+The service displays the system properties for a specific host.
+The `inMaintenance()` condition determines the system properties are returned only if you
```

Comment:
I would say, "determines [that] the system properties..."

Reply:
Fixed